app/src/main/res/layout/item_content_cart.xml

```diff
@@ -61,64 +66,64 @@
-                            android:layout_height="wrap_content"
-                            android:padding="@dimen/dp_5"
-                            android:text="@{viewModel.quantity}"
-                            android:textSize="@dimen/sp_18"
-                            />
-                        <View
-                            android:layout_width="@dimen/dp_1"
-                            android:layout_height="match_parent"
-                            android:layout_margin="@dimen/dp_5"
-                            android:background="@color/cardview_dark_background"
-                            />
-                        <ImageButton
-                            android:id="@+id/button_up_quantity"
-                            android:layout_width="wrap_content"
-                            android:layout_height="wrap_content"
-                            android:layout_gravity="center"
-                            android:background="@android:color/transparent"
-                            android:onClick="@{()-> viewModel.upQuantity()}"
-                            android:padding="@dimen/dp_5"
-                            android:src="@drawable/ic_add_black"
-                            />
-                    </LinearLayout>
+                        android:gravity="center"
+                        android:paddingTop="@dimen/dp_5"
+                        android:text="@{viewModel.orderTime}"
+                        android:textColor="@android:color/black"
+                        android:textSize="@dimen/sp_15"
+                        android:textStyle="italic"
+                        />
                 </LinearLayout>
-                <ImageButton
-                    android:id="@+id/button_delete_product"
-                    android:layout_width="wrap_content"
+                <ImageView
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
                     android:layout_gravity="center"
-                    android:layout_marginRight="@dimen/dp_20"
-                    android:background="@android:color/transparent"
-                    android:onClick="@{()-> viewModel.deleteProduct()}"
-                    app:srcCompat="@drawable/ic_delete"
+                    android:layout_weight="1.15"
+                    android:onClick="@{()-> viewModel.clickIconWarning()}"
+                    android:src="@drawable/ic_warning"
+                    android:visibility="@{viewModel.productTimeOut ? View.VISIBLE : View.GONE}"
+                    tools:ignore="ContentDescription"
                     />
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:layout_weight="1"
+                    android:gravity="center"
+                    android:orientation="vertical"
+                    android:paddingStart="@dimen/dp_10"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 14) you should *also* add `android:paddingLeft="@dimen/dp_10"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.